### ISSUE TITLE
What does v3 "breaking change" mean?

### ISSUE THREAD

**Issue by bmulholland** [nuxt-community/google-fonts-module#98]
v3 has "support for nuxt 3" listed as a "breaking change," but there's no indication of what breaks or how to fix it. Is it actually a breaking change? For what? What action is needed to upgrade?

**Comment by Skyost** [comment]
`there's no indication of what breaks`

Well, support for Nuxt 2 I think.

**Comment by bmulholland** [comment]
The PR in question has explicit handling for Nuxt 2: https://github.com/nuxt-community/google-fonts-module/pull/94/files#diff-030fc083b2cbf5cf008cfc0c49bb4f1b8d97ac07f93a291d068d81b4d1416f70R51

**Comment by josuemartinezz** [comment]
If you're not using nuxt v3, please install `"@nuxtjs/google-fonts": "^2.0.0"`

**Comment by bmulholland** [comment]
@josuemartinezz Can you please give more details? How do you know that? Why is it important. I see that you aren't a contributor/maintainer. If that restriction is indeed in place, then it should be _very clearly_ called out in the README, release notes, and so on.

**Comment by joserocha3** [comment]
@bmulholland Nuxt v3 is a rewrite of v2, therefore the underlying technologies are different and require different methods to get things to _work_. You can research [how to migrate from v2 to v3](https://v3.nuxtjs.org/migration/overview) or lookup the general differences, but this repo is not really the place to list all those out.

TLDR; no Nuxt v2 module work with Nuxt v3, they have to be rewritten.

**Comment by bmulholland** [comment]
@joserocha3 That's incorrect: @nuxt/kit, which the PR you linked to uses, [explicitly supports](https://github.com/nuxt/framework/blob/main/packages/kit/src/compatibility.ts#L24) | [Nuxt 2](https://github.com/nuxt/framework/blob/main/packages/kit/src/compatibility.ts#L72). And, in fact, I've used the latest version in production with both nuxt 2 and nuxt bridge, so unless I'm missing something, Nuxt 2 is still supported.

**Comment by jillesca** [comment]
> If you're not using nuxt v3, please install `"@nuxtjs/google-fonts": "^2.0.0"`

thanks [josueayala27](https://github.com/josueayala27), this should be on the documentation when installing the plugin. I spent some hours looking at the issue, until I found your comment.

**Comment by ricardogobbosouza** [comment]
Hi @bmulholland 
The v3 version of this module should work with nuxt 2 and 3

The main reason it is a "break change" is that its default settings have different values
v2 -https://github.com/nuxt-community/google-fonts-module/blob/v2.0.0/src/module.ts#L22-L39
v3 - https://github.com/nuxt-community/google-fonts-module/blob/main/src/module.ts#L23-L40
